net/wireguard/src/opnsense/service/conf/actions.d/actions_wireguard.conf

```diff
@@ -39,3 +39,9 @@ command:/usr/local/bin/wg show all latest-handshakes
 parameters:
 type:script_output
 message:show Wireguard handshakes
+
+[showhandshake.widget]
```

Comment:
this doesn't work, showhandshake points to the action above, configd doesn't support further nesting. have you tried this locally?

On my end this crashes configd.

Reply:
I have seen something similar ([action](https://github.com/opnsense/plugins/blob/master/sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf#L7) & [widget](https://github.com/opnsense/plugins/blob/master/sysutils/smart/src/www/widgets/widgets/smart_status.widget.php#L34)) for the `smart widget`.

I have not tried this locally but I can rename it.

Reply:
@x-jokay which does explain this report as well https://github.com/opnsense/plugins/pull/1848 (and missing root cause analysis). The smart widget is flawed as well, leading to an unstable `configd` service.

@darkain ^^ and there's your root cause analysis.

Reply:
:wink:

Reply:
Renamed to `[widget]`.